backend/src/packages/business/business.repository.ts

```diff
@@ -105,7 +105,7 @@ class BusinessRepository implements IRepository {
     const [item] = await this.db
       .driver()
       .update(this.businessSchema)
-      .set(payload)
+      .set({ ...payload, updatedAt: new Date() })
```

Comment:
may be we don't need it now as business table field updatedAt have defaultNow()  property

Reply:
I have just tested it, and without manually updating `updatedAt` field it is not updated by itself. defaultNow() means that the initial (default) value for `updatedAt` in a new row, if not specified, will be defaultNow, but it doesn't affect existing rows.